source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -233,6 +233,46 @@ number of writes may differ between source and destination.
 Transactions might not appear atomically on the destination cluster.
 Retryable writes may not be retryable on the destination cluster.
 
+.. _c2c-verifier-disclaimer:
+
+Embedded Verifier Disclaimer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting in 1.9, ``mongosync`` includes an embedded verifier to
+perform a series of verification checks on all supported
+collections on the destination cluster to confirm that
+it was successful in transferring documents from the
+source cluster to the destination.
+
+When you start the ``mongosync`` process, it provides a
+disclaimer advising the user that the verifier is enabled
+by default.
+
+.. code-block:: none
+
+   Embedded verification is enabled by default with compatible beta features.
+   Oplog Rollover Resilience and migrations with source clusters running MongoDB 5.0+
+   are the only compatible beta features. Verification checks for data consistency
+   between the source and destination clusters. Verification will cause mongosync
+   to fail if any inconsistencies are detected, but it does not check for all possible
+   data inconsistencies. It is your responsibility to review the documentation at
+   https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/verification/embedded
+   to see what data the embedded verifier checks, and verify any other data yourself.
+   Verification requires approximately 0.5 GB of memory per 1 million documents on the
+   source cluster and will fail if insufficient memory is available. Accepting this
+   disclaimer indicates that you understand the limitations, requirements, and failure
+   modes outlined in the documentation linked above. 
+
+   To disable the embedded verifier, specify 'verification: false' when starting mongosync. Please see
+   https://www.mongodb.com/docs/cluster-to-cluster-sync/current/reference/verification/
+   for alternative verification methods.
+
+   Do you want to continue? (y/n): 
+
+To skip this notification, start ``mongosync`` with the
+:option:`--acceptDisclaimer` option.
```

Comment:
Can we also add the `--acceptDisclaimer` option to the configurations https://deploy-preview-339--docs-cluster-to-cluster-sync.netlify.app/reference/configuration/#options

Reply:
Fixed